### PR TITLE
fix #115

### LINKED ISSUE #115: Bug [Export default]
## Version

1.0.6, 1.0.7
## Reproduction

Update from 1.0.5 to 1.0.6 or 1.0.7.
## Steps to reproduce

Normal use
## Expected Behavior

Render div or another component
## Actual Behavior

I hope these screenshots enough to understand the problem =)

### PR DESCRIPTION
**Pull request by stephenkingsley**
fix #115

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -18,6 +18,19 @@ import ThemeProvider from './models/ThemeProvider'
...
+const styledComponent = {}
+Object.keys(styled)
+  .map(ele => (styledComponent[ele] = styled[ele]))
+
+styledComponent.css = css
+styledComponent.keyframes = keyframes
+styledComponent.injectGlobal = injectGlobal
+styledComponent.ThemeProvider = ThemeProvider
+// export default
+export default styledComponent
```
Comment (k15a) [inline src/index.js]: With that you're breaking named exports.

```js
import {
  css,
  keyframes,
  injectGlobal,
  ThemProvider,
} from 'styled-components'
```

You now have to write

```js
import styled from 'styled-components'
const {
  css,
  keyframes,
  injectGlobal,
  ThemProvider,
} = styled
```

Note that this is not the same. For further information look at http://stackoverflow.com/questions/33524696/es6-destructuring-and-module-imports

**Comment by stephenkingsley** [comment]
oh, I think this is not my want. I will modify it as soon as possible.

**Comment by stephenkingsley** [comment]
oh, I think this is not my want. I will modify it as soon as possible.

**Comment by k15a** [comment]
You probably don't have to because that is a bug from webpack. See https://github.com/webpack/webpack/issues/3168 for further information. ;)

**Comment by stephenkingsley** [comment]
I think this bug just show in use <script />, not in commonJS. so I close

**Pull request closed without merging**